skill/__init__.py

```diff
@@ -24,3 +24,4 @@
 from .intent import WeatherIntent
 from .weather import CURRENT, DAILY, DailyWeather, HOURLY, WeatherReport
 from .util import LocationNotFoundError
+from .ovosbackend import OVOSApiBackend
```

Comment:
can this file be renamed? using backend will be confusing since ovos local backend exists

maybe call it just ovos api?

Reply:
this has been solved, renamed it to ovos api service